### PR TITLE
docs: Add contributors

### PR DESCRIPTION
**Pull request by vogelino**
This PR adds the contributors following the all-contributors spec.

There are two people missing in my opinion:

1. Tobias (no github account?)
2. Lea Scheidt (no github account?)

We might decide to ignore them though. Lea is already quoted as the owner of the illustration rights.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "projectName": "CityLAB Slides",
```
Comment (dnsos) [inline .all-contributorsrc:2]: ServiceAgent:innen?

**Review by dnsos — CHANGES_REQUESTED** [review]
Probably not a big deal, but there's still a wrong project name in the `.all-contributorsrc`.

> We might decide to ignore them though. Lea is already quoted as the owner of the illustration rights.

We can discuss more how we wanna handle non-GitHub contributors. For now I agree that it's fine the way it is.

File: .all-contributorsrc (hunk 1 above)
Comment (vogelino) [reply .all-contributorsrc]: Good catch! The name here should be the repo name (I just discovered this and might need to change that in the oder repos as well... :( ). So it is `service-agentinnen`

**Review by dnsos — APPROVED** [review]
👍

**Pull request merged**